assets/components/adFreeSection/AdFreeSection.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+// ----- Imports ----- //
+import React from 'react';
+import GridImage from 'components/gridImage/gridImage';
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+
+export default function AdFreeSection() {
+
+  return (
+    <LeftMarginSection modifierClasses={['blue']}>
+      <div className="component-ad-free">
+        <div className="component-ad-free__content">
+          <h3 className="component-ad-free__badge">New</h3>
```

Comment:
Because this is a shared component, we don't necessarily know what level of heading it should have, it depends on the context in the page. You could use the `Heading` component and pass down the level of heading in the hierachy to this component (e.g. `1`, `2`, `3` etc.).